src/components/Facepile/Facepile.Props.ts

```diff
@@ -19,11 +19,11 @@ export interface IFacepileProps extends React.Props<Facepile> {
   /** Button properties for the add face button */
   addButtonProps?: IButtonProps;
 
-  /** Button properties for the chevron button */
-  chevronButtonProps?: IButtonProps;
```

Comment:
removing props is a bad idea; if someone is using this, you just created a compiler error.

Instead, you should mark it as `@deprecated` and if possible try to redirect it.